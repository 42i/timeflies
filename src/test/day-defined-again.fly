
day 2012-11-07 8 12

day 2012-11-08 8 13

day 2012-11-07 8 14