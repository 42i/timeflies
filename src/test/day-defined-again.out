day-defined-again.fly:6: ERROR : day 2012-11-07 redefined.
